platform.mk

```diff
@@ -249,7 +249,7 @@ PRODUCT_PACKAGES += \
 # QCOM Bluetooth
 PRODUCT_PACKAGES += \
     android.hardware.bluetooth@1.0-impl \
-    android.hardware.bluetooth@1.0-service-qti
+    android.hardware.bluetooth@1.0-service
```

Comment:
Fwiw we can use the `1.1` loader, instead of the `1.0` loader. `android.hardware.bluetooth@1.0-impl` exposes a `1.1` service.

Reply:
Does not work, 
Lets worry about fixing the bugs instead of uprev-ing

Reply:
Let's not introduce bugs when making these dubious changes :wink:

Reply:
Dubious?
Did you even test this patch?
No wonder why people don't contribute anymore to sodp.

Reply:
android.hardware.bluetooth@1.0-impl-qti is a non existant package since, I dont know how long and bluetooth didnt work since android 12 and no one  cared to look into it due to lack of contributors, and when someone did try to fix, you call them dubious?

@jerpelea , Please let me know if you also think the same I will stop contributing
If you don't think so, please stop this "MR I KNOW EVERYTHING" stop reviewing arrogantly. 

The way he reviews. I believe he thinks(oh $hit, why didn't I identify this and fixed it) and tries to pin point unnecessary things.

Reply:
@ShujathMohd I appreciate that you take time to investigate the missing feature. 
I prefer to have a commit message describing the change so that we are less confused when we review the changes.
the commit message should contain the reason for the change and, in this case, the fact that you tested the BT functionality.

Reply:
> Did you even test this patch?

I don't test patches that have lacking or incomplete commit messages: it shows that the author didn't think it worth to spend time _justifying_ change, _or_ doesn't understand the subject.

You did a much better job in https://github.com/sonyxperiadev/device-sony-lena/pull/29/commits/bda09e7ee413c6789ffb29f9d57e28509c4aa69b (but please apply the same approach to the first commit) even though it is mostly a copy-paste of what I explained in Slack.